events/renderers.py

```diff
@@ -12,3 +26,198 @@ class JSONLDRenderer(JSONRenderer):
     media_type = 'application/ld+json'
     format = 'json-ld'
     charset = 'utf-8'
+
+
+class DOCXRenderer(renderers.BaseRenderer):
+    media_type = 'application/vnd.openxmlformats-officedocument.wordprocessingml.document'
+    format = 'docx'
+    charset = None
+    render_style = 'binary'
+
+    def get_query_params(self, renderer_context, param_name):
+        return renderer_context['request'].query_params.get(param_name, '')
+
+    def get_any_language(self, dictionary, default='fi', language_codes=None):
+        if not language_codes:
+            # Default order for when a language isn't found.
+            language_codes = list(set([default, 'fi', 'sv', 'en']))
+
+        for language_code in language_codes:
+            content = dictionary.get(language_code)
+            if content:
+                return content
+        return ''
+
+    def get_date_range(self, renderer_context):
```

Comment:
Hmm, it smells weird that a _renderer_ should read query parameters, and this duplicates the logic already in `_filter_event_queryset()`... Would it be possible to just have the viewset pass the days on to the renderer instead?